src/libcore/macros.rs

```diff
@@ -563,11 +563,11 @@ macro_rules! unimplemented {
 
 /// A macro to create an array of [`MaybeUninit`]
 ///
-/// This macro constructs and uninitialized array of the type `[MaybeUninit<K>; N]`.
+/// This macro constructs an uninitialized array of the type `[MaybeUninit<K>; N]`.
 ///
 /// [`MaybeUninit`]: mem/union.MaybeUninit.html
 #[macro_export]
-#[unstable(feature = "maybe_uninit", issue = "53491")]
+#[unstable(feature = "maybe_uninit_slice", issue = "53491")]
```

Comment:
I'd move this to a 4th feature gate `uninitialized_array` since I think this should never be stabilized in favor of https://github.com/rust-lang/rust/issues/49147 instead.

Reply:
I feel we can do that when/if the array stuff gets closer to being stabilized. Also, isn't https://github.com/rust-lang/rust/issues/49147 still a year off or so? (I heard it is blocked on removing the AST borrowck, and I also heard that that is about a year off.)

Reply:
Well... we could implement it for Rust 2018 only so it might be possible to ship earlier if we want to.  I'm mostly worried about packaging these into gates together and increasing the likelihood of shipping something that will become tech debt later. If we add another feature gate for the macro, we can always decide to change our minds later.

Reply:
I see no way we'd miss that during the stabilization discussion, but whatever. Done.

Reply:
Thanks ❤️